pkg/resource/deploy/step.go

```diff
@@ -1312,7 +1312,15 @@ func (s *RefreshStep) Apply() (resource.Status, StepCompleteFunc, error) {
 			s.Deployment().Diag().Warningf(diag.RawMessage(s.URN(), msg))
 		}
 	}
-	outputs := refreshed.Outputs
+
+	logging.V(10).Infof("Refreshed resource ID: %q, Inputs: %v, Outputs: %v",
```

Comment:
Hrm inputs and outputs can often contain secrets, no?  Should this be `V(11)`?

Reply:
I'll check but I think there are other places we print secrets below 11 for resource properties, but also we normally just print the property count which would be fine for here.